### PR TITLE
Fixing few typos

### PR DESCRIPTION
**Pull request by yuriskinfo**
HI, just fixing couple of typos in docs.
Thanks,
Yuri

### CODE REVIEW

File: docs/curl-config.1 [hunk 1]
```diff
@@ -74,8 +74,8 @@ to link your application with libcurl statically. (Added in 7.17.1)
 Outputs version information about the installed libcurl.
 .IP "--vernum"
 Outputs version information about the installed libcurl, in numerical mode.
-This outputs the version number, in hexadecimal, with 8 bits for each part;
-major, minor, patch. So that libcurl 7.7.4 would appear as 070704 and libcurl
+This outputs the version number, in hexadecimal, with 8 bits for each part:
+major, minor, and patch. So that libcurl 7.7.4 would appear as 070704 and libcurl
```
Comment (emilengler) [inline docs/curl-config.1:78]: For correct grammar you need to remove the comma before the and

Comment (jzakrzewski) [reply docs/curl-config.1:78]: it depends on how does one feel about the oxford comma... (a war similar to tabs vs spaces ;) )

Comment (yuriskinfo) [reply docs/curl-config.1:78]: Haha, indeed, for me serial/Oxford comma is automatic, but is actually a matter of style.  
https://en.wikipedia.org/wiki/Serial_comma

Comment (emilengler) [reply docs/curl-config.1:78]: Then my english teacher probably failed somewhere...
Anyway, thanks!

Comment (jzakrzewski) [reply docs/curl-config.1:78]: Ne, that's a nuance. Some don't know about it, some don't care, and some wage holly wars about it. I myself use it (hopefully correctly) but don't care if others do.

**Comment by bagder** [comment]
Thanks!

**Pull request closed without merging**